docs/data/material/migration/migrating-to-v6/migrating-to-v6.md

```diff
@@ -1,17 +1,37 @@
-# Migrating to v6
+# Upgrade to v6
 
-<p class="description">This guide explains how and why to migrate from Material UI v5 to v6.</p>
+<p class="description">This guide explains why and how to upgrade from Material UI v5 to v6.</p>
+
+## Why you should upgrade to Material UI v6
+
+### React Server Component support
+
+Material UI v6 introduces Pigment CSS, a zero-runtime CSS-in-JS styling engine to replace Emotion and styled-components as a more future-proof solution for writing styles in React 19 and beyond.
+With Pigment CSS, styles are extracted at build time rather than runtime, avoiding client-side recalculations and unlocking React Server Component (RSC) compatibility.
+This also leads to significant reductions in bundle sizes for Material UI apps.
+
+**In v6, Pigment CSS migration is opt-in.**
```

Comment:
I think removing "migration" makes it clear enough and connects better with the next sentence: Pigment CSS is the subject, and not the migration itself.

```suggestion
**In v6, Pigment CSS is opt-in.**
```

Reply:
Done